setup.cfg

```diff
@@ -0,0 +1,82 @@
+[flake8]
+ignore =
+    # whitespace before ':'
+    E203,
+    # too many leading ### in a block comment
+    E501,
```

Comment:
This should be `E266`, `E501` is "line too long".

Reply:
My bad I actually merged #612 where this was introduced... :cry:

Reply:
I'll rebase and fix it.